src/components/AlertCard/styles.module.css

```diff
@@ -154,17 +242,49 @@ div.row {
 }
 
 .Fyi {
-  background-color: #09b702;
+  width: 2rem;
+  height: 1.5625rem;
+  background: url(./low.svg);
+  background-size: 2.8125rem 2.1875rem;
+  position: relative;
+  bottom: 0.8rem;
+  padding: 0;
+  margin-left: -0.75rem;
+  margin-right: 0;
 }
 
 .Low {
```

Comment:
In general, it's custom to camelCase css class names. Can you lower case this?

Reply:
fixed